docs/examples/networking/route_table/route_table.tf

```diff
@@ -17,12 +13,25 @@ provider "oci" {
   region = "${var.region}"
 }
 
-resource "oci_core_route_table" "route_table1" {
+resource "oci_core_virtual_network" "ExampleVCN" {
+  cidr_block = "10.1.0.0/16"
+  compartment_id = "${var.compartment_ocid}"
+  display_name = "TFExampleVCN"
+  dns_label = "tfexamplevcn"
+}
+
+resource "oci_core_internet_gateway" "ExampleIG" {
+  compartment_id = "${var.compartment_ocid}"
+  display_name = "TFExampleIG"
+  vcn_id = "${oci_core_virtual_network.ExampleVCN.id}"
+}
+
+resource "oci_core_route_table" "ExampleRouteTable" {
```

Comment:
Should we start making samples use the default route table and dhcp options? The VCN may introduce default DHCP option and security list settings that Terraform user is not aware of and isn't defined in their TF config. These defaults may also change in the future. Maybe we should make it a recommended practice to start defining the defaults so that nothing is left to chance.

Reply:
I think that likely makes sense. Updating the examples and best practices will be the next step.